.github/workflows/ci-mlir.yml

```diff
@@ -104,6 +104,14 @@ jobs:
         export PATH=$PATH:${GITHUB_WORKSPACE}/llvm-project/build/bin/
         lit -v tests/filecheck/ -DCOVERAGE
 
+    - name: Run benchmarks
+      run: |
+        cd xdsl
+        # Add mlir-opt to the path
+        export PATH=$PATH:${GITHUB_WORKSPACE}/llvm-project/build/bin/
+        chmod u+x ./benchmarks/dialects/experimental/stencil/run-laplace-benchmark.sh
```

Comment:
Do you really need to do this on each run? Isn't one locally and then recommitting the file enough?